tvb_framework/tvb/core/services/project_service.py

```diff
@@ -109,7 +109,8 @@ def store_project(self, current_user, is_create, selected_id, **data):
         if started_operations > 0:
             raise ProjectServiceException("A project can not be renamed while operations are still running!")
         if is_create:
-            current_proj = Project(new_name, current_user.id, data["description"])
+            current_proj = Project(new_name, current_user.id, data["max_operation_size"], data["description"],
+                                   data["disable_imports"])
```

Comment:
Maybe this part could be re-written into something like bellow (as I see some tests failing from this area)
```
            max_size = data["name_size"] if "max_size" in data else None
            disable_imports = data["disable_imports"] if "disable_imports" in data else False
            current_proj = Project(new_name, current_user.id, max_size, data["description"], disable_imports)
```

Reply:
Having the validators implemented should ensure that these keys are always present in the data dict, even if they have None or False values.
In this case, shouldn't we adjust the tests to prepare the data dict in the same manner?